ci/scripts/integration_spark.sh

```diff
@@ -47,8 +42,12 @@ pushd ${spark_dir}
   build/mvn -B -DskipTests package
 
   # Run pyarrow related Python tests only
+  # "pyspark.sql.tests.arrow.test_arrow_grouped_map" and
+  # "pyspark.sql.tests.arrow.test_arrow_cogrouped_map" currently fail.
```

Comment:
Are they known issues? If so, could you add an issue number?

Reply:
I've added a comment with the issue: https://github.com/apache/arrow/issues/44986
 As shared on the new issue those tests were never run in the past but might be worth adding them as they are arrow related.